src/test/java/org/pesmypetcare/webservice/builders/PathTest.java

```diff
@@ -102,32 +104,36 @@ public void buildForumNamePath() {
     @Test
     public void buildForumNamesCollectionPath() {
         String path = Path.ofCollection(Collections.forumsNames, groupName);
-        assertEquals("groups_names/" + groupName + "/forums", path,
-            "Should return the path to the collectionof forum names.");
+        assertEquals("groups_names/" + groupName + "/forums", path
+            , "Should return the path to the collectionof forum names.");
     }
 
     @Test
     public void buildPetPath() {
         String path = Path.ofDocument(Collections.pets, userId, petName);
-        assertEquals("users/" + userId + "/pets/" + petName, path, "Should return the path to the pet.");
+        assertEquals("users/" + userId + "/pets/" + petName, path
+            , "Should return the path to the pet.");
     }
 
     @Test
     public void buildPetsCollectionPath() {
         String path = Path.ofCollection(Collections.pets, userId);
-        assertEquals("users/" + userId + "/pets", path, "Should return the path to the pets collection.");
+        assertEquals("users/" + userId + "/pets", path
+            , "Should return the path to the pets collection.");
     }
 
     @Test
     public void buildMemberPath() {
         String path = Path.ofDocument(Collections.members, groupId, userId);
-        assertEquals("groups/" + groupId + "/members/" + userId, path, "Should return the path to the member.");
+        assertEquals("groups/" + groupId + "/members/" + userId, path
+            , "Should return the path to the member.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [',' is preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)